docs/02-for-app-authors/02-requirements.md

```diff
@@ -305,9 +305,20 @@ pip etc.
 
 These manifests must be included in the submission.
 
+### Patches
+
+Sometime building requires some patches (see [above](#patches)). In
+that case these patches should be included along with the manifest.
```

Comment:
maybe merge https://docs.flathub.org/docs/for-app-authors/requirements#patches it with this to reduce duplicate headers.